### PR TITLE
Remove one container

### PR DESCRIPTION
**Pull request by chao007**
@liangxia Please help merge this pr.

### CODE REVIEW

**Comment by liangxia** [comment]
Have you tested your change ?

**Comment by chao007** [comment]
> Have you tested your change ?

Upload the new one.

File: storage/hostpath/security/hostpath.yaml [hunk 1]
```diff
@@ -8,27 +8,18 @@ spec:
...
-  - name: b
-    image: aosqe/hello-openshift
+  - image: aosqe/hello-openshift
+    name: a
     ports:
       - containerPort: 8081
         protocol: "TCP"
     securityContext:
-      runAsUser: 1000130002
+    runAsUser: 1000130001
```
Comment (liangxia) [inline storage/hostpath/security/hostpath.yaml]: Did you test before you summit/push ?

Comment (chao007) [reply storage/hostpath/security/hostpath.yaml]: yes

Comment (chao007) [reply storage/hostpath/security/hostpath.yaml]: uid is 1000130001
btw, I will remove part of steps in cucumber scripts

**Pull request merged**